go.mod

```diff
@@ -1,9 +1,9 @@
 module mittens
 
 require (
```

Comment:
Can you please run `go mod tidy` to clean up the `go.mod` and `go.sum` files?
You need to have Go modules enabled which you can do with `GO111MODULE=on`.

Reply:
Thanks!